app/controllers/users_controller.rb

```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
```

Comment:
Please add `show` and `index` actions to users controller.

Reply:
resources :users, only: %i[index show]
  resources :users_posts, only: %i[index show]

Is already there ... 

Please kind change to approved

Reply:
Fixed